test/interface_c.cpp

```diff
@@ -0,0 +1,182 @@
+
+// Example libexlixir
+const char * libelixir_path =
+  "../../LibTrixi.jl/examples/libelixir_p4est2d_dgsem_euler_sedov.jl";
+
+
+TEST(CInterfaceTest, JuliaProject) {
+
+    // be evil
+    const char * garbage = "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long"
+                           "this_string_is_just_way_toooooooooooooooooooo_long";
+    EXPECT_DEATH( trixi_initialize( garbage, "/tmp" ),
+                  "buffer size not sufficient for activation command");
+}
+
+
+TEST(CInterfaceTest, VersionInfo) {
+
+    // Initialize Trixi
+    trixi_initialize( julia_project_path, NULL );
+
+    // Check libtrixi version information
+    int major = trixi_version_library_major();
+    int minor = trixi_version_library_minor();
+    int patch = trixi_version_library_patch();
+    std::string version_string("");
+    version_string.append(std::to_string(major));
+    version_string.append(".");
+    version_string.append(std::to_string(minor));
+    version_string.append(".");
+    version_string.append(std::to_string(patch));
+    
+    EXPECT_STREQ(version_string.c_str(), trixi_version_library());
+
+    // Check julia packages version information
+    std::string version_string_julia(trixi_version_julia());
+    EXPECT_NE(version_string_julia.find("OrdinaryDiffEq"), std::string::npos);
+
+    std::string version_string_julia_ext(trixi_version_julia_extended());
+    EXPECT_NE(version_string_julia_ext.find("StartUpDG"), std::string::npos);
+
+    // Finalize Trixi
```

Comment:
```suggestion
    // Finalize libtrixi
```
? (to make it clear that we are initializing the library, not a simulation?)